packages/core/strapi/bin/strapi.js

```diff
@@ -299,7 +289,7 @@ program
     )
   )
   .addOption(new Option('-f, --file <file>', 'name to use for exported file (without extensions)'))
-  .addOption(excludeOption)
+  // .addOption(excludeOption)
```

Comment:
I think we can remove that if we're not using it. We can add it back easily if needed in the future.